paimon-core/src/main/java/org/apache/paimon/table/AbstractFileStoreTable.java

```diff
@@ -166,14 +166,43 @@ public Identifier identifier() {
 
     @Override
     public Optional<Statistics> statistics() {
-        // todo: support time travel
         Snapshot latestSnapshot = snapshotManager().latestSnapshot();
```

Comment:
Maybe just here to respect time travel options.

Reply:
Done